assets/panel/debugger.properties

```diff
@@ -405,6 +405,11 @@ breakpointMenuItem.removeCondition.label=Remove breakpoint condition
 breakpointMenuItem.removeCondition.accesskey=c
 breakpointMenuItem.editCondition.label=Edit breakpoint condition
 breakpointMenuItem.editCondition.accesskey=n
+breakpointMenuItem.disableAllAtLine.label=Disable breakpoints on line
+breakpointMenuItem.enableAllAtLine.label=Enable breakpoints on line
+breakpointMenuItem.removeAllAtLine.label=Remove breakpoints on line
+breakpointMenuItem.enableAllAtLine.accesskey=l
```

Comment:
Please move the accesskey close to its label, it avoids confusion for both localizers and developers (if they need to update something).